packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -581,8 +584,13 @@ mixin _CupertinoSheetRouteTransitionMixin<T> on PageRoute<T> {
     );
   }
 
+  /// Determines whether the content can be scrolled.
+  ///
+  /// If `true`, scrolling is enabled; otherwise, it remains fixed.
+  bool get enableDrag;
+
   /// Returns a [CupertinoSheetTransition].
-  static Widget buildPageTransitions<T>(
+  Widget buildPageTransitions(
```

Comment:
We can keep this as `static` if we add an enableDrag argument, then pass it in inside of `buildTransitions`. Keeping it static is more likely to help with performance.

Reply:
fix this commit
https://github.com/flutter/flutter/pull/163923/commits/9720184be95fea489b9fc7f8844eeebe12d65347